### PR TITLE
Templated AD/nonAD RadialReturnCreepStressUpdateBase

### PR DESCRIPTION
**Pull request by tophmatthews**
#16183

### CODE REVIEW

File: modules/tensor_mechanics/include/materials/ADRadialReturnCreepStressUpdateBase.h [hunk 1]
```diff
@@ -9,39 +9,4 @@
...
-protected:
-  virtual void initQpStatefulProperties() override;
-  virtual void propagateQpStatefulProperties() override;
-  virtual void computeStressFinalize(const ADRankTwoTensor & plastic_strain_increment) override;
-
-  /// Creep strain material property
-  ADMaterialProperty<RankTwoTensor> & _creep_strain;
-  const MaterialProperty<RankTwoTensor> & _creep_strain_old;
-};
+#include "RadialReturnStressUpdateBase.h"
```
Comment (tophmatthews) [inline modules/tensor_mechanics/include/materials/ADRadialReturnCreepStressUpdateBase.h:12]: Is it as simple as this @loganharbour ?

Comment (tophmatthews) [reply modules/tensor_mechanics/include/materials/ADRadialReturnCreepStressUpdateBase.h:12]: Crap...I misspelled...but you get the idea...

Comment (loganharbour) [reply modules/tensor_mechanics/include/materials/ADRadialReturnCreepStressUpdateBase.h]: Yep - I think that'll do it, as long as the class that existed there before is now defined in your new header.

**Comment by tophmatthews** [comment]
Ready for review @dschwen

**Review by dschwen — APPROVED** [review]

**Pull request merged**